roles/edpm_container_manage/tasks/shutdown.yml

```diff
@@ -27,17 +27,25 @@
       loop:
         - 'edpm-container-shutdown'
         - 'edpm-start-podman-container'
-    - name: "Create /usr/lib/systemd/system/edpm-container-shutdown.service"
+      when: not edpm_use_bootc
```

Comment:
I don't see `edpm_use_bootc` set anywhere. Is this a leftover from an earlier revision?

Reply:
Yeah, before the custom fact. I'll fix this